aten/src/ATen/native/AdaptiveAveragePooling3d.cpp

```diff
@@ -30,6 +31,7 @@ static void adaptive_avg_pool3d_out_frame(
     int64_t istrideH,
     int64_t istrideW) {
   int64_t d = 0;
+  at::internal::lazy_init_num_threads();
 #pragma omp parallel for private(d)
```

Comment:
oh, pragma omp creeped back into the codebase, we'll need to change this to parallel_for, but we don't have to do this in this PR